examples/harmony-library/README.md

```diff
@@ -33,7 +33,7 @@ module.exports = {
 ``` js
 return /******/ (function(modules) { // webpackBootstrap
 /******/ 	// The module cache
-/******/ 	var installedModules = {};
+/******/ 	let installedModules = {};
```

Comment:
Did you run the code example and this was the result?

Reply:
im not sure what you mean by result? but no i did not run that code locally here, but its only a simple remake to get rid of old syntax that cant break the code

Reply:
How can it break the code?
webpack generated JS have to be ES5 complaint, otherwise it won't work on older browsers.